src/components/Fullscreen.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FC, useState, useEffect} from 'react';
+import Box from './Box';
+
+const Fullscreen: FC = ({children}) => {
+	const [size, setSize] = useState({
+		columns: process.stdout.columns,
```

Comment:
You are assuming `process.stdout`, but that could be changed in the options: https://github.com/vadimdemedes/ink#stdout-1

Reply:
Good call, I noticed that should have sorted it - is there a mechanism by which the terminal is already captured in a context or anything?  If so, this seems like it may be a deeper dive than this "bare minimum".  If you have any tips, please!!! :D  If not, I'll look into it over the weekend.

Reply:
.. Okay I was too curious, so I walked through the renderer code there's a StdinContext, StdoutContext I can look at.